src/test_application/VehicleBuilder.cpp

```diff
@@ -24,6 +24,8 @@
  */
 #include "VehicleBuilder.h"
 
+#if 0
```

Comment:
you've basically deleted the whole file?

Reply:
yep. Most of the functions should remain the same once vehicles are re-implemented.